step-ca/templates.mdx

```diff
@@ -150,6 +152,95 @@ You can also import values
 from the `"templateData"` object
 in your provisioner's [configuration block](#configuring-the-ca-to-use-templates).
 
+### ASN.1 Values
+
+The X.509 templating system includes functions for ASN.1 encoding in a template.
+This can be useful for populating custom OID `extensions`:
+
+```
+"extensions": [
+  {"id": "1.2.3.4", "critical": true, "value": {{ asn1Enc "int:3848281" | toJson }}}
```

Comment:
Usually critical will be `false` on custom extensions. If the client/server does not understand that extension it should fail.
```suggestion
  {"id": "1.2.3.4", "critical": false, "value": {{ asn1Enc "int:3848281" | toJson }}}
```